source/operations/concepts/scanner.rst

```diff
@@ -0,0 +1,103 @@
+.. _minio-concepts-scanner:
+
+==============
+Object Scanner
+==============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Overview
+--------
+
+MinIO uses the built-in scanner to check objects for healing and to take any scheduled object actions.
+Such actions may include:
+
+- calculate data usage on drives
+- evaluate and apply configured :ref:`lifecycle management <minio-lifecycle-management>` or :ref:`object retention <minio-object-retention>` rules
+- perform :ref:`bucket <minio-bucket-replication>` or :ref:`site <minio-site-replication-overview>` replication
+- check objects for missing or corrupted data or parity shards and perform :ref:`healing <minio-concepts-healing>`
+
+The scanner performs these functions at two levels: cluster and bucket.
+At the cluster level, the scanner splits all buckets into groups and scans one group of buckets at a time.
+The scanner starts with any new buckets added since the last scan, then randomizes the scanning of other buckets.
+The scanner completes checks on all bucket groups before starting over with a new set of scans.
+
+At the bucket level, the scanner groups items in buckets and scans selected items from that bucket.
+The scanner selects objects for a scan based on a hash of the object name.
+Over a span of 16 scans, MinIO checks every object in the namespace.
+MinIO fully scans any prefixes known to be new since the last scan.
+
+Scan Length
+-----------
+
+Multiple factors impact the time it takes for a scan to complete.
+
+Some of these factors include:
+- Type of drives provided to MinIO
+- Throughput and :abbr:`iops <input/output operations per second>` available
```

Comment:
This shows in the page with the angle brackets, is that intended?

Reply:
supposed to be `( )` instead of `< >`